percona-server-9.1.yaml

```diff
@@ -168,15 +168,14 @@ test:
       runs: |
         # Create a non-root user for the MySQL test...
         adduser -D -h /home/mysql mysql
-        chown -R mysql:mysql /var/lib/mysql /run/mysqld /var/tmp
         mkdir -p /run/mysqld /var/lib/mysql /var/tmp
         chown -R mysql:mysql /run/mysqld /var/lib/mysql /var/tmp
     - name: Initialize MySQL data directory
       runs: |
         su mysql -c "mysqld --initialize-insecure --datadir=/var/lib/mysql" || exit 1
     - name: Start MySQL server with no password validation
       runs: |
-        su mysql -c "mysqld --datadir=/var/lib/mysql --socket=/run/mysqld/mysqld.sock --skip-grant-tables &"
+        su mysql -c "mysqld --datadir=/var/lib/mysql --socket=/run/mysqld/mysqld.sock --skip-grant-tables > /dev/null 2> /dev/null &"
```

Comment:
Rather than /dev/null, could we pipe this into a file called `percona-server.log`? It'll keep it consistent with other packages that have had redirects added